diff --git a/helpers/build_packages.sh b/helpers/build_packages.sh
index 29ae338..498d656 100755
--- a/helpers/build_packages.sh
+++ b/helpers/build_packages.sh
@@ -285,19 +285,19 @@ if [ "$BUILDMW" = "1" ]; then
             buildmw -u "https://github.com/sailfishos/sensorfw.git" \
                     -s rpm/sensorfw-qt5-hybris.spec || die
         fi
-        if [ $android_version_major -le 7 ]; then
-            buildmw -u "https://github.com/mer-hybris/geoclue-providers-hybris" \
-                    -s rpm/geoclue-providers-hybris.spec || die
-        fi
-        # build kf5bluezqt-bluez4 if not yet provided by Sailfish OS itself
-        sdk-assistant maintain $VENDOR-$DEVICE-$PORT_ARCH zypper se kf5bluezqt-bluez4 > /dev/null
-        ret=$?
-        if [ $ret -eq 104 ]; then
-            buildmw -u "https://github.com/sailfishos/kf5bluezqt.git" \
-                    -s rpm/kf5bluezqt-bluez4.spec || die
-            # pull device's bluez4 configs correctly
-            sdk-assistant maintain $VENDOR-$DEVICE-$PORT_ARCH zypper remove bluez-configs-mer
-        fi
+       if [ $android_version_major -le 7 ]; then
+           buildmw -u "https://github.com/mer-hybris/geoclue-providers-hybris" \
+                   -s rpm/geoclue-providers-hybris.spec -N|| die
+       fi
+       # # build kf5bluezqt-bluez4 if not yet provided by Sailfish OS itself
+       # sdk-assistant maintain $VENDOR-$DEVICE-$PORT_ARCH zypper se kf5bluezqt-bluez4 > /dev/null
+       # ret=$?
+       # if [ $ret -eq 104 ]; then
+       #     buildmw -u "https://github.com/sailfishos/kf5bluezqt.git" \
+       #             -s rpm/kf5bluezqt-bluez4.spec || die
+       #     # pull device's bluez4 configs correctly
+       #     sdk-assistant maintain $VENDOR-$DEVICE-$PORT_ARCH zypper remove bluez-configs-mer
+       # fi
     fi
 fi
 
